CDCgov/prime-reportstream
# 10207: add CDC elims metabase queries to source control

This PR adds the queries written for the CDC ELIMS team to source control

Test Steps:
1. Go to https://staging.prime.cdc.gov/metabase/collection/49-cdc-elims-collection and check out all the queries

**As a note @arnejduranovic  has already tested these as part of figuring out the functionality**

## Changes
- Adds the queries for the CDC ELIMS team

## Checklist

### Testing

### Process

## Linked Issues
- #10207 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
